packages/utils/src/configured_bignumber.ts

```diff
@@ -11,4 +11,22 @@ BigNumber.config({
     DECIMAL_PLACES: 78,
 });
 
+// Set a debug print function for NodeJS
+// Upstream issue: https://github.com/MikeMcl/bignumber.js/issues/188
+import isNode = require('detect-node');
+if (!isNode) {
```

Comment:
Ack, this should be `if (isNode) {` right?

Reply:
Yes :)